etc/kayobe/pulp.yml

```diff
@@ -405,6 +413,16 @@ stackhpc_pulp_repository_rpm_repos:
     state: present
     required: "{{ stackhpc_pulp_sync_for_local_container_build | bool and stackhpc_pulp_sync_el_8 | bool }}"
 
+  - name: CentOS Stream 8 - Ceph Quincy
```

Comment:
Unsure how we've ended up adding Ceph Quincy for CS8 and CS9?

Reply:
It was [added to release train](https://github.com/stackhpc/stackhpc-release-train/commit/47f89613f80dbc83438cb9fa29f33507f11dc782#diff-79b98dc6a02dff7f0a526062916901d18f3002f6aa8d5f274aae46483f75f40eR156-R160) so I thought I'd add it in here too.

Reply:
hmm, missed that in the review

Reply:
Do you want it included here?

Reply:
@g0rgamesh do you know why Ceph Qunicy was added for CS8 in your RL9 patch?

Reply:
It must be a leftover from times when I was confused about adding C8S repos at some point in the conversation. Should I remove them?

Reply:
I think we can remove it.

Reply:
I've removed it from this config